Code/Engine/RendererVulkan/Device/Implementation/SwapChainVulkan.cpp

```diff
@@ -347,7 +350,8 @@ ezResult ezGALSwapChainVulkan::DeInitPlatform(ezGALDevice* pDevice)
   DestroySwapChainInternal(pVulkanDevice);
   if (m_vulkanSurface)
   {
-    pVulkanDevice->DeleteLater(m_vulkanSurface);
+    ezVulkanAllocation context = reinterpret_cast<ezVulkanAllocation>(m_WindowDesc.m_pWindow);
```

Comment:
I really shouldn't abuse the **ezVulkanAllocation** handle to piggy back my window ptr, will add a context in another push.